easybuild/easyconfigs/b/binutils/binutils-2.25.eb

```diff
@@ -18,12 +18,19 @@ builddependencies = [('zlib', '1.2.8')]
 preconfigopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
 prebuildopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
 
-# make sure that system libraries are also considered by ld
-configopts = '--with-sysroot=/'
+# make sure that system libraries are also considered by ld and ld.gold is also built
+configopts = '--with-sysroot=/ --enable-gold --enable-ld=default '
 
-binlist = ['addr2line', 'ar', 'as', 'c++filt', 'elfedit', 'gprof', 'ld', 'ld.bfd', 'nm',
-           'objcopy', 'objdump', 'ranlib', 'readelf', 'size', 'strings', 'strip' ]
+# Ensure PIC
+buildopts = 'CFLAGS="-fPIC"'
+
+# If you require libiberty uncomment the following
+#postinstallcmds = ["cp libiberty/libiberty.a %(installdir)s/lib/; cp include/libiberty.h %(installdir)s/include/",
```

Comment:
libiberty is built by default but not included in the installation (says in configure `# Note that libiberty is not a target library`). Libiberty source is hosted in GCC and that's the correct place to enable the library but on the mailing lists people said that they might have a non-PIC GCC build